lib/actions/preload.coffee

```diff
@@ -50,9 +50,11 @@ getApplicationsWithSuccessfulBuilds = (resin, deviceType) ->
 				build.status == 'success'
 		applications
 
-selectApplication = (expectedError, resin, form, deviceType) ->
+selectApplication = (expectedError, resin, form, deviceType, applicationInfoSpinner) ->
```

Comment:
I'm not sure moving these to `utils` right now would make sense.
If someday someone wants to reuse this function, he/she can move it to utils.
For example utils already has a `selectApplication` function but I couldn't reuse it because the app_name instead of the application itself.

I'll go with the simple solution: requiring resin-cli-visuals in selectApplication.